Source/WTF/wtf/FileSystem.cpp

```diff
@@ -336,7 +336,7 @@ MappedFileData::MappedFileData(const String& filePath, MappedFileMode mapMode, b
     closeFile(fd);
 }
 
-#if HAVE(MMAP)
+#if HAVE(MMAP) && !OS(AMIGAOS)
```

Comment:
just wondering what the reason is for this

Reply:
This is a little bit problematic for us now. There were some discussions in amigans for it. Have a look at 
https://www.amigans.net/modules/newbb/viewtopic.php?post_id=133143&keywords=mmap

Reply:
afxgroup said he implemented these functions. Is it not possible to remove this macro and find out if it works?

Reply:
@3246251196 You were right. mmap works with the new clib2, so I removed the above check in my last commi